### PR TITLE
ncu: add page

### PR DESCRIPTION
**Pull request by Clumsy-Coder**
## what
- add tldr page for npm package `npm-check-updates`
  - how to install package 
    - `npm install -g node-check-updates`
  - how to list oudated packages in current folder
    - `ncu` - how to list outdated global packages
    - `ncu -g` 
  - how to upgrade all packages in current folder 
    - `ncu -u && npm install` 
  - how to interactively upgrade npm package in current folder 
    - `ncu -i && npm install` 
  - how to view `ncu` options 
    - `ncu -h`

## why
- useful command for listing outdated npm packages

## where
- ./pages/common/ncu.md

## usage
tldr -u
tldr ncu

---

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/ncu.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# ncu
+
+> npm plugin for `npm-check-updates`.
+> Used for checking outdated npm packages locally or globally.
+> More information: <https://github.com/raineorshine/npm-check-updates>.
+
+- Install npm package node-check-updates globally:
```
Comment (kbdharun) [inline pages/common/ncu.md:7]:
```suggestion
- Install npm package `node-check-updates` globally:
```

File: pages/common/ncu.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+`npm install -g node-check-updates`
+
+- List outdated npm packages in current folder:
```
Comment (kbdharun) [inline pages/common/ncu.md:11]:
```suggestion
- List outdated npm packages in the current directory:
```

File: pages/common/ncu.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`ncu`
+
+- List outdated global npm packages:
+
+`ncu -g`
+
+- Upgrade all npm packages in current folder:
```
Comment (kbdharun) [inline pages/common/ncu.md:19]:
```suggestion
- Upgrade all npm packages in the current directory:
```

File: pages/common/ncu.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+`ncu -u && npm install`
+
+- Interactively upgrade npm package in current directory:
+
+`ncu -i && npm install`
+
+- View `ncu` options:
```
Comment (kbdharun) [inline pages/common/ncu.md:27]:
```suggestion
- Display help:
```

File: pages/common/ncu.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+- List outdated global npm packages:
+
+`ncu -g`
+
+- Upgrade all npm packages in current folder:
+
+`ncu -u && npm install`
+
+- Interactively upgrade npm package in current directory:
```
Comment (kbdharun) [inline pages/common/ncu.md:23]:
```suggestion
- Interactively upgrade npm packages in the current directory:
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and Thanks for your contribution. I have some suggestions for this page.

File: pages/common/ncu.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+# ncu
+
+> npm plugin for `npm-check-updates`.
```
Comment (kbdharun) [inline pages/common/ncu.md:3]:
```suggestion
> NPM plugin for `npm-check-updates`.
```

File: pages/common/ncu.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+> Used for checking outdated npm packages locally or globally.
```
Comment (kbdharun) [inline pages/common/ncu.md:4]:
```suggestion
> Find newer versions of package dependencies and check outdated npm packages locally or globally.
```
How about this?

**Review by kbdharun — APPROVED** [review]

File: pages/common/ncu.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+# ncu
+
+> Find newer versions of package dependencies and check outdated npm packages locally or globally.
+> More information: <https://github.com/raineorshine/npm-check-updates>.
+
+- Install npm package `node-check-updates` globally:
+
+`npm install -g node-check-updates`
```
Comment (acuteenvy) [inline pages/common/ncu.md:8]: Is there any other page with an example of how to install the program?
```suggestion
```

File: pages/common/ncu.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+`ncu -g`
+
+- Upgrade all npm packages in the current directory:
+
+`ncu -u && npm install`
+
+- Interactively upgrade npm packages in the current directory:
+
+`ncu -i && npm install`
```
Comment (acuteenvy) [inline pages/common/ncu.md:24]:
```suggestion
- Upgrade all dependencies in the current directory:

`ncu -u`

- Interactively upgrade dependencies in the current directory:

`ncu -i`
```

File: pages/common/ncu.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
+# ncu
+
+> Find newer versions of package dependencies and check outdated npm packages locally or globally.
+> More information: <https://github.com/raineorshine/npm-check-updates>.
```
Comment (acuteenvy) [inline pages/common/ncu.md:4]:
```suggestion
> Find newer versions of package dependencies and check outdated npm packages locally or globally.
> `ncu` only updates dependency versions in `package.json`. To install the new versions, run `npm install` afterwards.
> More information: <https://github.com/raineorshine/npm-check-updates>.
```

File: pages/common/ncu.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+- Install npm package `node-check-updates` globally:
+
+`npm install -g node-check-updates`
+
+- List outdated npm packages in the current directory:
```
Comment (acuteenvy) [inline pages/common/ncu.md:10]:
```suggestion
- List outdated dependencies in the current directory:
```

File: pages/common/ncu.md (hunk 8 above)
Comment (kbdharun) [reply pages/common/ncu.md:8]: Yep, We don't generally include installation commands to a page.

**Review by kbdharun — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]
